components/fuse/src/main/java/org/datacleaner/components/fuse/CoalesceFunction.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * DataCleaner (community edition)
+ * Copyright (C) 2014 Neopost - Customer Information Management
+ *
+ * This copyrighted material is made available to anyone wishing to use, modify,
+ * copy, or redistribute it subject to the terms and conditions of the GNU
+ * Lesser General Public License, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+package org.datacleaner.components.fuse;
+
+import java.util.List;
+
+class CoalesceFunction {
+
+    private final boolean _considerEmptyStringAsNull;
+
+    public CoalesceFunction(boolean considerEmptyStringAsNull) {
+        _considerEmptyStringAsNull = considerEmptyStringAsNull;
+    }
+
+    public Object coalesce(List<? extends Object> values) {
+        for (final Object value : values) {
+            if (value != null) {
+                if (_considerEmptyStringAsNull) {
+                    if (!"".equals(value)) {
```

Comment:
I first wanted to protest about not using String.isEmpty(), but this avoids a cast, and is only a little less readable in itself, so :+1: